Document how to inject env variables from ConfigMaps into Function (#9985)
kyma-project/kyma: docs/serverless/05-01-serverless-envs.md

type: Configuration
---

> Configuration of Function's environment variables.

Reviewer: ```suggestion
You can configure environment variables either separately for a given runtime or make them runtime-agnostic using a ConfigMap.
```

> ## Use ConfigMap's key-values to define Function's environment variables.

Reviewer: ```suggestion
## Define environment variables in a ConfigMap
```

> See example of a Function using the value of the `my-var` key, stored in `my-vars-cm` ConfigMap, as `MY_VAR` environment variable.

Reviewer: ```suggestion
ConfigMaps allow you to define Function's environment variables for any runtime through key-value pairs. After you define the values, simply reference a given ConfigMap in the Function custom resource (CR) through the **valueFrom** parameter. See an example of such a Function CR that specifies the `my-var` value as a reference to the key stored in the `my-vars-cm` ConfigMap as the `MY_VAR` environment variable.
```
